getaround_utils/spec/getaround_utils/mixins/loggable_spec.rb

```diff
@@ -85,7 +92,7 @@ def append_infos_to_loggable(payload)
       it 'inject the class name' do
         expect(dummy_logger).to receive(:info)
           .with('message="dummy" key="value" origin="ChildClass"')
-        subject.use_loggable(:info, 'dummy', key: :value)
+        subject.loggable_log(:info, 'dummy', key: :value)
```

Comment:
This should actually still be `use_loggable` which is defined in this spec, and verifies that it works when called (used) inside a class method.

Reply:
🤦‍♂ 
Of course !!! My bad